src/System.Threading/tests/MutexTests.cs

```diff
@@ -165,45 +172,182 @@ private static IEnumerable<string> GetNamePrefixes()
         public static IEnumerable<object[]> AbandonExisting_MemberData()
         {
             var nameGuidStr = Guid.NewGuid().ToString("N");
-            for (int waitType = 0; waitType < 2; ++waitType) // 0 == WaitOne, 1 == WaitAny
+            for (var waitType = WaitHandleWaitType.WaitOne; waitType <= WaitHandleWaitType.WaitAll; ++waitType)
```

Comment:
`foreach (WaitHandleWaitType waitType in Enum.GetValues(typeof(WaitHandleWaitType)))`?